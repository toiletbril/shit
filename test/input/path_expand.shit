echo data/*;
echo data/*/;
echo data/*/*;
echo data/*/*/;
echo data/*g;
echo data/abcd*;
echo data/??*fg;
echo data/[a-b]*/;
echo data/*[^g];
echo data/.*;