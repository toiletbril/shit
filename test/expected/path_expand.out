[Sequence]
  [SequenceNode Simple]
    [Exec "echo data/*"]
  [SequenceNode Simple]
    [Exec "echo data/*/"]
  [SequenceNode Simple]
    [Exec "echo data/*/*"]
  [SequenceNode Simple]
    [Exec "echo data/*/*/"]
  [SequenceNode Simple]
    [Exec "echo data/*g"]
  [SequenceNode Simple]
    [Exec "echo data/abcd*"]
  [SequenceNode Simple]
    [Exec "echo data/??*fg"]
  [SequenceNode Simple]
    [Exec "echo data/[a-b]*/"]
  [SequenceNode Simple]
    [Exec "echo data/*[^g]"]
  [SequenceNode Simple]
    [Exec "echo data/.*"]
data/abcd data/abcdefg data/fourth data/hello-d data/hello+K
data/abcd/ data/abcdefg/
data/abcd/first data/abcd/second data/abcdefg/hjkl data/abcdefg/third
data/abcdefg/hjkl/
data/abcdefg
data/abcd data/abcdefg
data/abcdefg
data/abcd/ data/abcdefg/
data/abcd data/fourth data/hello-d data/hello+K
data/.fifth
[Code 0]
